db/main.go

```diff
@@ -156,39 +171,43 @@ func getCandidateFiles(fileList *drive.FileList) map[string]gDriveCandFiles {
 }
 
 func getDBItems(candFiles map[string]gDriveCandFiles, googleDriveService *drive.Service) (map[string]*votingCity, error) {
+	log.Printf("files to retrieve protocol buffers: %d\n", len(candFiles))
 	dbItems := make(map[string]*votingCity)
 	for _, c := range candFiles {
-		content, err := func() ([]byte, error) {
-			response, err := googleDriveService.Files.Get(c.candidatureFile.Id).Download()
+		if c.candidatureFile != nil {
+			content, err := func() ([]byte, error) {
+				response, err := googleDriveService.Files.Get(c.candidatureFile.Id).Download()
+				if err != nil {
+					return nil, fmt.Errorf("falha ao pegar bytes de arquivo de candidatura, erro %q", err)
+				}
+				defer response.Body.Close()
+				b, err := ioutil.ReadAll(response.Body)
+				if err != nil {
+					return nil, fmt.Errorf("falha ao ler bytes de arquivo de candidatura, erro %q", err)
```

Comment:
Seguindo o idioma Go: encurtando nome sem piorar a legibilidade. 

```suggestion
				r, err := googleDriveService.Files.Get(c.candidatureFile.Id).Download()
				if err != nil {
					return nil, fmt.Errorf("falha ao pegar bytes de arquivo de candidatura, erro %q", err)
				}
				defer r.Body.Close()
				b, err := ioutil.ReadAll(r.Body)
				if err != nil {
					return nil, fmt.Errorf("falha ao ler bytes de arquivo de candidatura, erro %q", err)
```